### DISCUSSION TITLE
Question/Request

### DISCUSSION THREAD

**Discussion by iwasthere83** [scfmod/FS22_TerraFarm#49] [Ideas]
Hello, I was wondering if in the future you guys planned to add separate LUA that could add our own configuration file to tie into the configuration folder inside of TerraFarm. I was able to clone one inside of it to name my own to get my implement working with TerraFarm. I feel like it would make the requests of "Can you please add this mod" taken off your plate and put it back on the modders. Examples in pictures
![image](https://user-images.githubusercontent.com/45979447/152371292-6ad05e44-1f67-417e-8d04-0f2e8f5f42d2.png)
![image](https://user-images.githubusercontent.com/45979447/152371363-7bcdeb37-d241-4d3d-912d-aa3b2eef7735.png)

**Comment by scfmod** [comment]
The functionality has been implemented since first release on github if I remember correctly, You can put yer custom configuration file in modSettings/TerraFarm/configurations/  :-)

In the new update there will be some changes to how configurations works, whether loading from modSettings will work on release or later is still TBD.

**Reply by iwasthere83** [reply]
So I can add that folder path inside my mod with my config xml and TerraFarm will load it? Or am I miss understandnig?

**Reply by scfmod** [reply]
Meant in your FarmingSimulator2022/modSettings/TerraFarm/configurations/ :-)
Loading from mod files won't be supported yet due to project being beta and configuration structures are subject to be changed at any time as features are added/changed.

**Comment by iwasthere83** [comment]
That works so thank you!

**Comment by Dark300653** [comment]
Bonjour,

J'ai installé le mod terrafarm dans le dossier mod de mon jeux, hors, peut importe la carte que je lance, je n'y est pas accès, est-ce qu'il y a une manip à faire ?

**Comment by Loanwolf1** [comment]
Are there any dump trucks or dump trailers that are enabled in TerraFarm. It would be really nice to be able to dump and have it add to the ground as in real life or to dump material as usual.
